_data/events.yml

```diff
@@ -161,8 +138,8 @@
   githubUrl: ""
   blogPost: ""
 
-- name: "QA minded: code complete?"
-  by: ""
+- name: "Testing Track: Test Containers"
+  by: "Simon Verhoeven"
   subTitle: ""
```

Comment:
Maybe put here as subtitle:

"Hands-on Test Containers and other test-related exercises"